### DISCUSSION TITLE
Get list of live flows?

### DISCUSSION THREAD

**Discussion by petsneakers** [mitmproxy/mitmproxy#6210] [General]
Not seeing this in the docs but figured I'd ask anyway.  Is it possible in an addon to get a list of currently connected flows?  Using `mitdump`.

**Comment by mhils** [comment]
For mitmproxy and mitmweb you could use the `view.flows.resolve` command, for mitmdump you need to keep track of active flows yourself.

**Comment by petsneakers** [comment]
Thanks for confirming.
